src/Sandbox.php

```diff
@@ -451,9 +451,9 @@ public function setConfigurationId(string $configurationId): self
         return $this;
     }
 
-    public function getConfigurationId(): ?string
+    public function getConfigurationId(): string
     {
-        return $this->user;
```

Comment:
To jsem zvedavej kde vsude se to pouzivalo. V sandbox komponente nastesti ne.

Reply:
Jestli to nekdo nekdy nekde takle fakt pouzil, tak si zaslouzi, aby se to ted rozbilo :smile: